torch/nn/modules/dropout.py

```diff
@@ -184,6 +184,46 @@ def forward(self, input: Tensor) -> Tensor:
         return F.alpha_dropout(input, self.p, self.training)
 
 
+class FeatureDropout(_DropoutNd):
+    r"""Randomly masks out entire channels (a channel is a feature map,
+    e.g. the :math:`j`-th channel of the :math:`i`-th sample in the batch input
+    is a tensor :math:`\text{input}[i, j]`) of the input tensor).
+
+    Each channel will be zeroed out independently on every forward call with
+    probability :attr:`p` using samples from a Bernoulli distribution.
+
+    Internally used by `DropoutXd` making stack traces less confusing.
```

Comment:
I guess this may be omitted from the main docs or may be moved to some notes section

Reply:
Sure! Removed